src/main/java/se/sundsvall/messaging/api/model/request/SnailMailRequest.java

```diff
@@ -62,5 +61,9 @@ public record Attachment(
 
         @ValidBase64
         @Schema(description = "The attachment (file) content as a BASE64-encoded string", example = "aGVsbG8gd29ybGQK", requiredMode = REQUIRED)
-        String content) { }
+        String content,
+
+        @Schema(description = "The envelope type for the letter", nullable = true)
```

Comment:
Guess the case where envelopeType is null is handled in snail-mail-sender? I.e. what's the expected behaviour when it's left out?

Reply:
Yes, it's handled in snail-mail.